### PR TITLE
Add SUSE Rancher to ADOPTERS.md

### PR DESCRIPTION
**Pull request by aiyengar2**
## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue._

Just documenting that Rancher uses Prometheus Operator!

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
- Added SUSE Rancher to ADOPTERS.md
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -195,6 +195,22 @@ Details (optional):
...
+
+Uses [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus): Yes
+
+Rancher currently maintains patches on the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) Helm chart to support use cases for Prometheus Operator across various different cluster types and setups (including Windows clusters) that are managed via the Rancher product. All Rancher users that install Monitoring V2 deploy this chart. 
+
+For more information, please see https://rancher.com/docs/rancher/v2.6/en/monitoring-alerting/how-monitoring-works/.
+
+Rancher's open-source patched Helm chart, `rancher-monitoring`, is maintained in the `release-v2.*` branches of [rancher/charts](https://github.com/rancher/charts).
+
+Rancher's legacy Helm chart, also called `rancher-monitoring`, is maintained in the `release-v2.*` branches [rancher/system-charts](https://github.com/rancher/system-charts); while this chart also uses Prometheus Operator, it is independent of [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack).
```
Comment (jiaqiluo) [inline ADOPTERS.md:212]:
```suggestion
Rancher's legacy Helm chart, also called `rancher-monitoring`, is maintained in the `release-v2.*` branches of [rancher/system-charts](https://github.com/rancher/system-charts); while this chart also uses Prometheus Operator, it is independent of [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack).
```

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

File: ADOPTERS.md [hunk 2]
```diff
@@ -195,6 +195,22 @@ Details (optional):
 - 25000 samples/s
 - 1.2M active series
 
+## SUSE Rancher
+
+[suse.com/products/suse-rancher](https://www.suse.com/products/suse-rancher/)
+
+Environments: RKE, RKE2/k3s, AWS, Azure, Google Cloud, Bare Metal, etc.
```
Comment (bashofmann) [inline ADOPTERS.md:202]: Should be `K3s` instead of `k3s`.

Comment (cbron) [reply ADOPTERS.md:202]:
```suggestion
Environments: RKE, RKE2, K3s, AWS, Azure, Google Cloud, Bare Metal, etc.
```

File: ADOPTERS.md [hunk 3]
```diff
@@ -195,6 +195,22 @@ Details (optional):
...
+
+Uses [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus): Yes
+
+Rancher currently maintains patches on the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) Helm chart to support use cases for Prometheus Operator across various different cluster types and setups (including Windows clusters) that are managed via the Rancher product. All Rancher users that install Monitoring V2 deploy this chart. 
+
+For more information, please see https://rancher.com/docs/rancher/v2.6/en/monitoring-alerting/how-monitoring-works/.
```
Comment (cbron) [inline ADOPTERS.md:208]:
```suggestion
For more information, please see [how Rancher monitoring works](https://rancher.com/docs/rancher/v2.6/en/monitoring-alerting/how-monitoring-works/).
```

File: ADOPTERS.md [hunk 4]
```diff
@@ -195,6 +195,22 @@ Details (optional):
...
+
+Rancher's open-source patched Helm chart, `rancher-monitoring`, is maintained in the `release-v2.*` branches of [rancher/charts](https://github.com/rancher/charts).
```
Comment (cbron) [inline ADOPTERS.md:210]: I'd say both of these paragraphs are a bit too much info. We could just add them as links.

Comment (aiyengar2) [reply ADOPTERS.md]: simplified

File: ADOPTERS.md [hunk 5]
```diff
@@ -195,6 +195,20 @@ Details (optional):
...
 
+## SUSE Rancher
+
+[suse.com/products/suse-rancher](https://www.suse.com/products/suse-rancher/)
+
+Environments: RKE, RKE2, K3s, Windows, AWS, Azure, Google Cloud, Bare Metal, etc.
+
+Uses [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus): Yes
+
+Rancher Monitoring supports use cases for Prometheus Operator across various different cluster types and setups that are managed via the Rancher product. All Rancher users that install Monitoring V2 deploy this chart.
```
Comment (paulfantom) [inline ADOPTERS.md:206]:
```suggestion
Rancher Monitoring supports use cases for Prometheus Operator across various different
cluster types and setups that are managed via the Rancher product. All Rancher users that
install Monitoring V2 deploy this chart.
```

Just a line limit thing.

File: ADOPTERS.md [hunk 6]
```diff
@@ -195,6 +195,20 @@ Details (optional):
...
+
+Environments: RKE, RKE2, K3s, Windows, AWS, Azure, Google Cloud, Bare Metal, etc.
+
+Uses [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus): Yes
+
+Rancher Monitoring supports use cases for Prometheus Operator across various different cluster types and setups that are managed via the Rancher product. All Rancher users that install Monitoring V2 deploy this chart. 
+
+For more information, please see [how Rancher monitoring works](https://rancher.com/docs/rancher/v2.6/en/monitoring-alerting/how-monitoring-works/).
+
+The open-source rancher-monitoring Helm chart (based on [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)) can also be found at [rancher/charts](https://github.com/rancher/charts).
```
Comment (aiyengar2) [inline ADOPTERS.md:210]:
```suggestion
The open-source rancher-monitoring Helm chart (based on [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)) can be found at [rancher/charts](https://github.com/rancher/charts).
```

**Review by paulfantom — APPROVED** [review]
lgtm on green

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Comment by paulfantom** [comment]
Awesome! Welcome SUSE Rancher to prometheus-operator adopters! :tada:

**Pull request merged**